data/migrations/20211102124951_inboxes.js

```diff
@@ -15,7 +15,7 @@ exports.up = (knex) => {
 
     .createTable('messages', function (table) {
       table.increments('id');
-      table.timestamp('created_at').defaultTo(knex.fn.now());
+      table.timestamp('sent_at').defaultTo(knex.fn.now());
```

Comment:
Good call on the naming here, I think this will make more sense from a FE perspective.